aries-core/src/js/components/core/Popover/Popover.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import { Box, Button, Drop } from 'grommet';
+import { Close } from 'grommet-icons';
+
+export const Popover = ({
+  a11yTitle,
+  title,
+  children,
+  footer,
+  targetRef,
+  onClose,
+  ...rest
+}) => {
+  return (
+    <Drop
+      round="xxsmall"
+      role="dialog"
+      elevation="medium"
+      stretch={false}
+      target={targetRef?.current}
+      onClose={onClose}
+      {...rest}
+    >
+      <Box gap="small" direction="row">
+        <Box flex gap="small" pad={{ vertical: 'small', left: 'small' }}>
+          {/* do we like this format of passing in
+           title children and footer? */}
+          {title}
+          {children}
+          {footer}
+        </Box>
+        {/* // see if we have a close t shirt size for the 8 px */}
+        {/* // have xsmall for right or small none cuts off focus   */}
+        <Box pad={{ right: 'xsmall', top: '8px' }}>
```

Comment:
Having a hard coded pixel value feels like it goes against our building principles.

I'd recommend we stick with "xsmall".

If we feel like our t-shirt scale isn't providing enough granularity, and we feel this kind of thing is a valid use case, it might lump in with this broader tokens discussion: https://github.com/grommet/hpe-design-system/issues/3906

Reply:
@KennyAtHPE